### DISCUSSION TITLE
Make this a one stop app for Remembering Relatives and their relation with you

### DISCUSSION THREAD

**Discussion by casual-coderr** [monicahq/monica#6225] [Ideas]
Hi,
I think most of the people would have had this moment in life, you go to a family party or wedding but you don't remember the person's relation to you. They might say i'm your uncle's co-father or something like that.
It'll be awkward and you won't even know if they are telling the truth or what to react there. 

Monica already has a great number of features by creating contacts, remembering their birthdays,  how you met them and lot more.

So if there is an option to create a mindmap of the contacts being created in monica and label them.
Eg.
![Screenshot 2022-07-15 at 1 17 36 PM](https://user-images.githubusercontent.com/93610399/179177442-a9c40575-b479-4ff1-985c-8f47b9c9dc89.png)

This is a big big feature to ask but will change this project forever since from what i have searched there is nothing like this in the whole internet..

**Comment by majant0** [comment]
If you're into genealogy, there are apps in that space that could do that for you. I use [TNG](https://www.tngsitebuilding.com) for this and I can pull up my website to do this exact thing. Not my website, but here is another website that uses this genealogy software and shows this type of relationship: [Relationship Map](https://holliergenealogy.info/relationship.php?altprimarypersonID=&savedpersonID=I17609&secondpersonID=I5545&maxrels=1&disallowspouses=0&generations=15&tree=Hollier-Dufilho&primarypersonID=I1279). In this example, the two people I randomly selected are 2nd cousins once removed.

That being said, this would still be a welcome feature on Monica.
